plugins/etymology.py

```diff
@@ -28,6 +28,8 @@ def etymology(text, reply):
     try:
         response.raise_for_status()
     except HTTPError as e:
+        if response.status_code == 404:
+            return 'No etymology found for {} :('.format(text)
```

Comment:
```suggestion
            return "No etymology found for {} :(".format(text)
```
Text presented to the user should be enclosed in double quotes